Lib/fontTools/cffLib/__init__.py

```diff
@@ -2310,6 +2304,7 @@ def decompileAllCharStrings(self, progress):
 				progress.increment(0)  # update
 			i = i + 1
```

Comment:
@miguelsousa I think i'll leave that old piece of code like that for now. I'm actually tempted to deprecate the progress bar interface altogether. BTW, do you know if anybody is using that? /cc @justvanrossum